Headlines/AppDelegate.swift

```diff
@@ -78,24 +80,36 @@ class AppDelegate: UIResponder,
             userSettingsManager.firstOpenDate = Date()
         }
         
-        let success: ([Topic]?) -> Void = { (topics) in
-            self.newsFetched = topics
+        let completion: ([Topic]?, [Category]?, NSError?) -> Void = { [unowned self] (topics, categories, error) in
+            if let e = error {
+                // #TODO: Replace this with an alert error in the future and a "Try again" button
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/b49c86a9b05bec3bbe17833b5a716a8293c42cf1/Headlines/AppDelegate.swift#L85"></span>TODOs should be avoided (Replace this with an alert err…).</td>
    </tr>
  </tbody>
</table>